sal_interfaces/MTM2/MTM2_Events.xml

```diff
@@ -101,4 +101,32 @@
     </item>
 </SALEvent>
 
+<SALEvent>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_logevent_isForceBalanceSystemOn</EFDB_Topic>
+    <Alias>isForceBalanceSystemOn</Alias>
+    <Description>Force balance (FB) system is on or off?</Description>
+    <item>
+        <EFDB_Name>status</EFDB_Name>
+        <Description>Ture if FB is on, False if off.</Description>
+        <IDL_Type>boolean</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALEvent>
+
+<SALEvent>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_logevent_isZenithAngleFromMount</EFDB_Topic>
```

Comment:
If you accept my proposal to rename the command to `selectInclinationTelemetrySource` I would recommend renaming this `inclinationTelemetrySource` and the payload would be changed to be the enumeration value instead of `isZenithAngleFromMount`.

Reply:
I like inclinationTelemetrySource better

Reply:
Done in commit: 9c5cc3f. Thanks!